requirements.txt

```diff
@@ -0,0 +1 @@
+odoo_autodiscover==1.0.3
```

Comment:
Needed because you installed your odoo addons with pip and odoo_autodiscover?

Reply:
My set up is:
- Vagrant box with Odoo installed under a custom directory (i.e. not in site-packages)
- 3 folders (in `/vagrant/data/` containing sets of modules appended to `addons_path` in config file

On running `pytest_odoo` initially it wasn't able to collect the tests as it was trying to import the tests under `data.[folder_in_config].[module_folder].tests.[test_name]`.

Not sure if it's just my setup but installing `odoo_autodiscover` made it work. Feel free to disregard if not standard setup.